docker/kubernetes-tentacle/scripts/configure-and-run.sh

```diff
@@ -245,6 +251,60 @@ function registerTentacle() {
     tentacle "${ARGS[@]}"
 }
 
+function addAdditionalServerInstancesIfRequired() {
+    if [[ -z "$ServerCommsAddresses" ]]; then
+      return;
+    fi
+
+    IFS=',' read -ra SERVER_ADDRESSES <<<"$ServerCommsAddresses"
+    len=${#SERVER_ADDRESSES[@]}
+
+    if [[ -z "$ServerCommsAddress" && len -eq 1 ]]; then
+      return;
+    fi
+
+    echo "Registering additional HA Servers..."
```

Comment:
It looks like this will allow registering additional instances even non HA?

Reply:
The way I have this setup, it uses the `ServerUrl`  to get the thumbprint and stuff, which points to the main server you're registering from. We could allow users to pass in different `ServerUrl`s for each Comms Address, but they would also need to pass in different API keys or bearer tokens for each one too. 🤔

Reply:
As discussed [here](https://octopusdeploy.slack.com/archives/C0738NKQN7K/p1716509518356089), I think we can continue with this change and add registering with separate server instances later if there is a user call for it.